packages/input-container/theme/lumo/vaadin-input-container-styles.js

```diff
@@ -8,7 +8,6 @@ registerStyles(
   'vaadin-input-container',
   css`
     :host {
-      border-radius: var(--lumo-border-radius-m);
       background-color: var(--lumo-contrast-10pct);
       padding: 0 calc(0.375em + var(--lumo-border-radius-m) / 4 - 1px);
```

Comment:
The padding calculation should now use the new `--vaadin-input-container-border-radius`

Reply:
Done.